[Serializer] Add `debug:serializer` command (#18211)
symfony/symfony-docs: serializer.rst

            $framework->serializer()->nameConverter('serializer.name_converter.camel_case_to_snake_case');
        };

Debugging the Serializer
------------------------

Use the ``debug:serializer`` command to dump the serializer metadata of a
given class:

.. code-block:: terminal

>     $ php bin/console debug:serializer 'App\Entity\SomeClass'

Reviewer: ```suggestion
    $ php bin/console debug:serializer 'App\Entity\Book'
```

>         App\Entity\SomeClass

Reviewer: ```suggestion
        App\Entity\Book
```

        --------------------

        +----------------+------------------------------------------------------------+
        | Property       | Options                                                    |
        +----------------+------------------------------------------------------------+

>         | firstArgument  | [                                                          |

Reviewer: Can we use something more real life instead of `firstArgument` ?

Maybe name and ISBN, because the groups also mention `book`

Thanks

Author: done

        |                |   "denormalizationContexts" => []                          |
        |                | ]                                                          |
        | secondArgument | [                                                          |
        |                |   "groups" => [                                            |
        |                |       "book:read",                                         |
        |                |   ]                                                        |
        |                |   "maxDepth" => null,                                      |
        |                |   "serializedName" => null                                 |
        |                |   "ignore" => false                                        |
        |                |   "normalizationContexts" => [],                           |
        |                |   "denormalizationContexts" => []                          |
        |                | ]                                                          |
        +----------------+------------------------------------------------------------+

.. versionadded:: 6.3

>     This debug command was introduced in Symfony 6.3.

Reviewer: ```suggestion
    The debug:serializer`` command was introduced in Symfony 6.3.
```
